File: 08 SoftwareTechnologies 06.17/Software-Technologies/java/01. Java-Syntax-Basic-Web-Calculator-Skeleton/src/main/java/com/softuni/Entity/Calculator.java
package com.softuni.Entity;

public class Calculator {

    private double leftOperand;

    private double rightOperand;

    private String operator;

    public Calculator(double leftOperand, double rightOperand, String opeator){
        this.leftOperand = leftOperand;
        this.rightOperand = rightOperand;
        this.operator = opeator;
    }

    public double getLeftOperand() {
        return leftOperand;
    }

    public void setLeftOperand(double leftOperand) {
        this.leftOperand = leftOperand;
    }

    public double getRightOperand() {
        return rightOperand;
    }

    public void setRightOperand(double rightOperand) {
        this.rightOperand = rightOperand;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public double calculateResult(){
        double result;

        switch (this.operator){
            case "+":
                result = this.leftOperand + this.rightOperand;
                break;
            case "-":
                result = this.leftOperand - this.rightOperand;
                break;
            case "*":
                result = this.leftOperand * this.rightOperand;
                break;
            case "/":
                result = this.leftOperand / this.rightOperand;
                break;
            default:
                result = 0;
        }

        return result;
    }

}

File: 08 SoftwareTechnologies 06.17/Software-Technologies/asp.net/01. Calculator/Calculator-CSharp/Models/Calculator.cs
namespace Calculator_CSharp.Models
{
    public class Calculator
    {
        public Calculator(decimal leftOperand, decimal rightOperand, string @operator)
        {
            LeftOperand = leftOperand;
            RightOperand = rightOperand;
            Operator = @operator;
            Result = 0;
        }

        public Calculator()
        {
            this.Result = 0;
        }

        public decimal LeftOperand { get; set; }

        public decimal RightOperand { get; set; }

        public string Operator { get; set; }

        public decimal Result { get; set; }


        public decimal CalculatoreResult()
        {
            decimal result = 0;

            switch (Operator)
            {
                case "+":
                    result = this.LeftOperand + this.RightOperand;
                    break;
                case "-":
                    result = this.LeftOperand - this.RightOperand;
                    break;
                case "*":
                    result = this.LeftOperand * this.RightOperand;
                    break;
                case "/":
                    result = this.LeftOperand / this.RightOperand;
                    break;
            }

            this.Result = result;

            return result;
        }
    }
}